### ISSUE TITLE
Test scalability of hyperparameters setting

### ISSUE THREAD

**Issue by Luka-Loncar** [masa-finance/masa-bittensor#100]
Linking this to task #77 for @5u6r054 to test at scale. 

Since we currently don't know if the provided implementation can support more than 1 node, we want to scale testing of the implementation. 

Action items:

- [ ] Create a test that allows spinning up more nodes in the subnet (e.g. 1000 "fake" nodes)
- [ ] Make sure that implementation is solid enough by running the test suite - otherwise create follow-up tasks

**Comment by juanmanso** [comment]
Out of scope for this week's planning. Would suggest to keep it on `No Status`

**Comment by juanmanso** [comment]
Since starting Q3, this enters the current scope. However, I think this is blocked by Docker images/Docker-compose to be able to have a testing environment. 

I'm open to suggestions that tell other opinions but as of now I think it's:
- Blocked by #85 
- Blocked by #109 
- [Maybe] Blocked by #111

**Comment by juanmanso** [comment]
> Since starting Q3, this enters the current scope. However, I think this is blocked by Docker images/Docker-compose to be able to have a testing environment.
> 
> I'm open to suggestions that tell other opinions but as of now I think it's:
> 
> * Blocked by [Set up subtensor, miner (with masa protocol worker) and validator using docker compose #85](https://github.com/masa-finance/masa-bittensor/issues/85)
> * Blocked by [ci: Container images release #109](https://github.com/masa-finance/masa-bittensor/issues/109)
> * [Maybe] Blocked by [Github action to deploy devnet (opentensor subtensor instance with a subnet) #111](https://github.com/masa-finance/masa-bittensor/issues/111)

Might still be blocked, however the competition could prove to be such test of hyperparameters. Would require more info on what's the expected output of this task. Maybe it is enough with the competition and we just need to define the output of this task rather than the steps to complete it.

Think this still needs triage by the @masa-finance/subnet team as a whole

**Comment by juanmanso** [comment]
> > Since starting Q3, this enters the current scope. However, I think this is blocked by Docker images/Docker-compose to be able to have a testing environment.
> > I'm open to suggestions that tell other opinions but as of now I think it's:
> > 
> > * Blocked by [Set up subtensor, miner (with masa protocol worker) and validator using docker compose #85](https://github.com/masa-finance/masa-bittensor/issues/85)
> > * Blocked by [ci: Container images release #109](https://github.com/masa-finance/masa-bittensor/issues/109)
> > * [Maybe] Blocked by [Github action to deploy devnet (opentensor subtensor instance with a subnet) #111](https://github.com/masa-finance/masa-bittensor/issues/111)
> 
> Might still be blocked, however the competition could prove to be such test of hyperparameters. Would require more info on what's the expected output of this task. Maybe it is enough with the competition and we just need to define the output of this task rather than the steps to complete it.
> 
> Think this still needs triage by the [@masa-finance/subnet](https://github.com/orgs/masa-finance/teams/subnet) team as a whole

+1

Suggest we close this since iteration over the subnet is being done on the fly by the needs of the environment. This might be already redundant or outdated by now

**Comment by Luka-Loncar** [comment]
Obsolete now since we are on the mainnet already, and scaling works.
